mothership-ec/cog-mothership-file-manager
# Remove namespace for HttpFoundation\File from File\Create class

#### What does this do?

Fixes issue that Union were having where they were getting the following error:
`Fatal Error (E_COMPILE_ERROR): Cannot use Symfony\Component\HttpFoundation\File as File because the name is already in use {"file":"/var/www/vhosts/unionmusicstore.com/production/releases/20140721104600/vendor/message/cog-mothership-file-manager/src/File/Create.php","line":14} []`

As far as I can see, the namespace in question isn't actually being used anywhere in the class and I tested uploading a file without it and it still worked so I'm gonna assume it's safe, but it's worth noting I couldn't recreate the issue on my machine
#### How should this be manually tested?

Check that you can upload an image, and double check that the namespace isn't being used
#### Related PRs / Issues / Resources?
#### Anything else to add? (Screenshots, background context, etc)
